retroshare-gui/src/gui/NewsFeed.cpp

```diff
@@ -74,6 +75,8 @@ NewsFeed::NewsFeed(QWidget *parent) :
     RsAutoUpdatePage(1000,parent),
     ui(new Ui::NewsFeed)
 {
+	rsEvents->registerEventsHandler( [this](std::shared_ptr<const RsEvent> event) { handleEvent(event); }, mEventHandlerId );
```

Comment:
Is mEventHandlerId initialized somewhere?

Reply:
I thought that registerEventsHandler() initializes the ID for you. It would actually make sense!